src/main/java/com/main/controller/PersonController.java

```diff
@@ -0,0 +1,29 @@
+package com.main.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.main.dto.ResponseDto;
+import com.main.service.PersonService;
+
+@RestController
+public class PersonController {
+	
+	private PersonService personService;
+	
+	public PersonController(PersonService personService) {
+		this.personService = personService;
+	}
+
+	@RequestMapping(value = { "/search", "/search/{word}" }, method = RequestMethod.GET)
+	public ResponseEntity<ResponseDto<String>> MessagesByPerson(@RequestHeader("user_id") long userId,
+			@PathVariable(required = false) String word) {
```

Comment:
@PathVariable shouldn't be used to filter/search, use @QueryParam instead.

Reply:
I think for this case is better pathvariable

Reply:
Why?

Reply:
I think wrong, PathVariable works better If Im looking by any Id, in other cases like searching with specific parameters of the search, It's better Queryparams. I will change this.